docs/README.md

```diff
@@ -18,32 +18,39 @@ Precondition: python3 and pip3 are already installed.
 1. Clone the library repo  
 `git clone https://github.com/kentik/community_sdk_python.git`
 
-1. `cd community_sdk_python/`
+1. `cd community_sdk_python/kentik_api_library/`
+
+1. Install Python dependencies  
+`python -m pip install --upgrade -r requirements.txt`  
+`python -m pip install --upgrade -r requirements-dev.txt`  
+
+1. Generate grpc client code  
+`python setup.py grpc_stubs`
 
 1. Install the library from repo  
-`pip3 install -e kentik_api_library/`
+`pip install -e .`
 
 1. Run unit tests  
-`pytest kentik_api_library/tests/unit/ kentik_api_library/tests/integration/`
+`python setup.py pytest`
 
 1. Run mypy  
-`mypy kentik_api_library/kentik_api kentik_api_library/tests/ kentik_api_library/examples/`
+`python setup.py mypy`
```

Comment:
BTW, as we're using also pylint on CI, how about adding it around here (python setup.py pylint)?

Reply:
python setup.py pylint is already available

Reply:
Yeah, I know. But we don't have usage of pylint listed here (and we have for all other tools that we use)

Reply:
pylint needs some config and code adjustments before it can be recommended here

Reply:
pylint configuration ongoing: https://github.com/kentik/community_sdk_python/pull/112

Reply:
@mateuszmidor The purpose of #112 is not to configure pylint. It only changes the way it is executed.